src/Java/Geovis/EarthSource.java

```diff
@@ -0,0 +1,98 @@
+import java.awt.BorderLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JPanel;
+import javax.swing.SwingUtilities;
+
+import vtk.vtkActor;
+import vtk.vtkEarthSource;
+import vtk.vtkNativeLibrary;
+import vtk.vtkPanel;
+import vtk.vtkPolyDataMapper;
+
+//Author: Bharatesh Chakravarthi,
+//Affiliation: VELab, Chung Ang University, Seoul, South Korea. 
+ 
+public class EarthSource extends JPanel implements ActionListener 
+{
+	  private static final long serialVersionUID = 1L;
+	  private vtkPanel renWin;
+	  private JButton exitButton;
+
+	  // -----------------------------------------------------------------
+	  // Load VTK library and print which library was not properly loaded
+	  static 
+	  {
+	    if (!vtkNativeLibrary.LoadAllNativeLibraries()) 
+	    {
+	      for (vtkNativeLibrary lib : vtkNativeLibrary.values()) 
+	      {
+	        if (!lib.IsLoaded()) 
+	        {
+	          System.out.println(lib.GetLibraryName() + " not loaded");
+	        }
+	      }
+	    }
+	    vtkNativeLibrary.DisableOutputWindow(null);
+	  }
+	  // -----------------------------------------------------------------
+  
+  
+	  public EarthSource() 
+	  {
+		    super(new BorderLayout());
+		    
+		    vtkEarthSource EarthSource = new vtkEarthSource();
+		    EarthSource.OutlineOff();
+		    //EarthSource.OutlineOn();
+		    EarthSource.Update();
+		    
+		    
+		    //Create a Mapper and Actor
+		    vtkPolyDataMapper Mapper = new vtkPolyDataMapper();
+		    Mapper.SetInputConnection(EarthSource.GetOutputPort());
+		
+		    vtkActor Actor = new vtkActor();
+		    Actor.SetMapper(Mapper);
+		    Actor.GetProperty().SetColor(1.0, 1.0, 1.0);
```

Comment:
Please use vtkNamedColors to set the colors of actors backgrounds etc. It helps the user in that something like "White" in this case is easier to understand then (1, 1, 1). They can also more easily change colors e.g. to "SandyBrown" to produce more pleasing color combinations. Check out the cxx examples for more info.

Reply:
Am unable to use vtkNamedColor member functions from the jar file I’ve created. Am working on in it to find out the ways to use vtkNamedColors in java.  
I’ve tried a lot and still figuring out to find solutions as am unable to fetch from library itself.